pygfx/objects/_base.py

```diff
@@ -77,3 +83,13 @@ def update_matrix_world(
         if update_children:
             for child in self._children:
                 child.update_matrix_world()
+
+    def look_at(self, target: Vector3):
```

Comment:
Oh sweet, so we can orient any object this way 😮

Reply:
yes! it also properly takes into account any parent transforms